include/dxc/dxcapi.h

```diff
@@ -124,7 +133,8 @@ typedef struct DxcShaderHash {
 #define DXC_PART_OUTPUT_SIGNATURE         DXC_FOURCC('O', 'S', 'G', '1')
 #define DXC_PART_PATCH_CONSTANT_SIGNATURE DXC_FOURCC('P', 'S', 'G', '1')
 
-// Some option arguments are defined here for continuity with D3DCompile interface
+// Some option arguments are defined here for continuity with D3DCompile
```

Comment:
Should these comment "//" markers be "///" like the others?

Reply:
No, these comments are only intended for the header file and so shouldn't be picked up by doxygen.